storybook/storybook-react/src/Button/Button.stories.tsx

```diff
@@ -11,20 +11,16 @@ const meta = {
   title: 'Buttons/Button',
   component: Button,
   args: {
-    children: 'Default',
+    children: 'Tekst',
```

Comment:
I'd use the text of the primary story here, no need to set text we'll never show imo

Reply:
I thought ‘Tekst’ would be the best global default for the `children` prop. I had been typing ‘Button label’, ‘Checkbox label’, ‘Form label’ here but 1 that’s English en 2. too much work.

Reply:
But we don't really need a global default, right? We always have a primary story, so the `children` prop can just have the primary story text, 'Primary' in this case.

Reply:
Alright, [updated](https://github.com/Amsterdam/design-system/pull/1086/commits/aaa891a90e4d8a7e1e9e37363c06f8649e7c3ce4) for 4 components.

Reply:
I think you misunderstood me, the primary button label should be 'Primary'

Reply:
Sorry, I disagree. I don’t want to mix up examples with documentation. Examples should display realistic content or an essential phrase; the variation of a component is to be inferred from the title and the props.

We could go for ‘Opslaan’, ‘Bewerken’ etc. if you prefer, although we should think about the best phrase for a link, a checkbox and a form label as well.